GossipHooks.cpp

```diff
@@ -116,22 +116,32 @@ bool Eluna::OnGossipHello(Player* pPlayer, Creature* pCreature)
 bool Eluna::OnGossipSelect(Player* pPlayer, Creature* pCreature, uint32 sender, uint32 action)
 {
     START_HOOK_WITH_RETVAL(CreatureGossipBindings, GOSSIP_EVENT_ON_SELECT, pCreature->GetEntry(), false);
+    auto original_menu = *pPlayer->PlayerTalkClass;
```

Comment:
to keep consistency with the other variable names, I'd rename this to `originalMenu`